addons/mail/static/src/models/discuss_public_view.js

```diff
@@ -60,6 +69,11 @@ registerModel({
         isChannelTokenSecret: attr({
             default: true,
         }),
+        messagingAsPublicView: one('Messaging', {
+            compute: '_computeMessagingAsPublicView',
```

Comment:
In 16.0, you can define compute inline of field declaration:
```js
compute() {
   return replace(this.messaging);
},
```

No need to define function in `recordMethods` :)

Reply:
I put
```js
compute() {
   return this.messaging;
},
```
Since the `replace` function no longer exists. It is done like that in other places of the code and it works so I guess that's fine.

Reply:
`replace` is indeed implicit in v16, good job 🙂